### ISSUE TITLE
notebook `declearn-with-sklearn.ipynb` fails in testing

### ISSUE THREAD

**Issue by mvesin** [fedbiomed/fedbiomed#910]
When running notebook `declearn-with-sklearn.ipynb` it fails in testing:

```
mse = np.mean((fed_model.predict((X_test-scaling_mean)/scaling_sd) - y_test)**2)

...

File ~/.conda/envs/fedbiomed-researcher/lib/python3.10/site-packages/sklearn/linear_model/_base.py:419, in LinearClassifierMixin.predict(self, X)

...

ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 784 is different from 6)
```

**Comment by ybouilla** [comment]
The only bug I have seen so far is in the TrainingPlan of the first regressor model ` dataset = pd.read_csv(self.dataset_path, delimiter=',')` that should be ` dataset = pd.read_csv(self.dataset_path, delimiter=';')`

The order of the models is misleading because the testing part should be run with the first model, but when we run the model cell by cell, we are testing the second model (which is a SGDclassifier) with the data of the first model, hence the matrix multiplication error you have spotted. Possible actions to change the notebook are discussed internally with the team and will be updated in this issue

**Comment by ybouilla** [comment]
Fixed in PR #911
